package.json

```diff
@@ -10,7 +10,8 @@
   },
   "devDependencies": {
     "eslint": "3",
-    "mocha": "3"
+    "mocha": "3",
+    "eslint-plugin-mocha": "4.9.0"
```

Comment:
@vladikoff I got that message too so I installed the plugin, then tested from there.